### PR TITLE
Update links to JIRA tickets.

### PR DESCRIPTION
**Pull request by jonathanknowles**
## Issue

None

## Details

This PR updates all links to Jira tickets across the whole repository.

These are now hosted at `cardanofoundation.atlassian.net` instead of `input-output-hk.atlassian.net`.

### CODE REVIEW

File: docs/site/src/design/concepts/eras.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The roadmap is actually divided into _phases_, under which one or more eras (har
 
 The mechanism whereby the Cardano Node operates in different modes - depending on which slot it is up to - is called the "Hard Fork Combinator".
 
-The following [seminars](https://input-output.atlassian.net/wiki/spaces/EN/pages/718962750/IOHK+Research+Engineering+Seminar) (company internal use only) are a good introduction:
+The following [seminars](https://cardanofoundation.atlassian.net/wiki/spaces/EN/pages/718962750/IOHK+Research+Engineering+Seminar) (company internal use only) are a good introduction:
```
Comment (HeinrichApfelmus) [inline docs/site/src/design/concepts/eras.md]:
```suggestion
The following [seminars](https://input-output.atlassian.net/wiki/spaces/EN/pages/718962750/IOHK+Research+Engineering+Seminar) (company internal use only) are a good introduction:
```

This is a link to the IOHK confluence, not our JIRA.

**Review by HeinrichApfelmus — APPROVED** [review]
One link is wrong, but the others look good to me. 🤓

**Comment by HeinrichApfelmus** [comment]
Jonathan, I think that merging without the requested change would not be ideal. 😅 Could you bring up our process around auto-merge and code review at the next team meeting, so that we clarify how to go about it?

**Comment by jonathanknowles** [comment]
> Jonathan, I think that merging without the requested change would not be ideal. 

Completely agree. I suspect that the automatic "merge when ready" may have bitten us here. I've applied your requested change (see 417815e821c477d826352d8a1654fb50898d2a9d) — thanks for spotting this!

> Could you bring up our process around auto-merge and code review at the next team meeting, so that we clarify how to go about it?

I've added it to the agenda. 👍🏻

File: docs/site/src/design/concepts/eras.md (hunk 1 above)
Comment (jonathanknowles) [reply docs/site/src/design/concepts/eras.md]: Fixed in: 417815e821c477d826352d8a1654fb50898d2a9d.

**Pull request merged**